src/Markdown.MAML/Transformer/ModelTransformerBase.cs

```diff
@@ -250,7 +250,30 @@ protected MamlInputOutput InputOutputRule()
                 return null;
             }
 
-            MamlInputOutput typeEntity = new MamlInputOutput()
+            MamlInput typeEntity = new MamlInput()
+            {
+                TypeName = headingNode.Text,
+                Description = SimpleTextSectionRule(),
+                Parameters = new List<MamlParameter>(),
```

Comment:
where is it populated? Can it be done closer to this line, so the logic is not spread apart and structures are immutable thru most of the code?